client/my-sites/backup/backup-date-picker/test/index.jsx

```diff
@@ -56,13 +61,14 @@ describe( 'BackupDatePicker', () => {
 
 	test( "Shows 'Yesterday' as the previous date if the current date is today", () => {
 		const today = moment();
+		render( <BackupDatePicker selectedDate={ today } onDateChange={ () => {} } /> );
 
-		const picker = shallow( <BackupDatePicker selectedDate={ today } onDateChange={ () => {} } /> );
-
-		const previousDate = picker.find(
-			'.backup-date-picker__select-date--previous .backup-date-picker__display-date'
+		const yesterdayTextNode = screen.getByText( 'Yesterday' );
+		expect( yesterdayTextNode ).toBeInTheDocument();
```

Comment:
`.toBeInTheDocument()` is generally fine, but for visible elements, I always recommend `.toBeVisible()`, as something can be in the DOM tree but be invisible to the user. And what really matters is whether the user can interact with it. 

This applies to a few more instances where we use `toBeInTheDocument()`.